templates/directorcontrol/emailblacklist.html

```diff
@@ -1,42 +1,47 @@
 $def with (entries)
-$:{RENDER("common/stage_title.html", ["Blacklist Email Address", "Director Control Panel", "/directorcontrol"])}
+$:{RENDER("common/stage_title.html", ["Edit Account Creation Email Blacklist", "Director Control Panel", "/directorcontrol"])}
 
 <div class="content blacklisted-emails">
-  <form method="POST" action="/directorcontrol/emailblacklist">
-    $:{CSRF()}
-    <table>
-      <thead>
-        <tr>
-          <th width="10%">Remove</th>
-          <th width="40%">Domain Name</th>
-          <th width="40%">Reason Blacklisted</th>
-          <th width="10%">Added by</th>
-        </tr>
-      </thead>
-      <tbody>
-        $for item in entries:
+  $if entries:
+    <form method="POST" action="/directorcontrol/emailblacklist">
+      $:{CSRF()}
+      <table>
+        <thead>
           <tr>
-            <td><input class="checkbox" type="checkbox" name="remove_selection" value="$item['id']"></td>
-            <td>$item['domain_name']</td>
-            <td>$item['reason']</td>
-            <td>$item['name']</td>
+            <th width="10%" style="text-align: left;">Remove</th>
+            <th width="40%" style="text-align: left;">Domain Name</th>
+            <th width="40%" style="text-align: left;">Reason Blacklisted</th>
+            <th width="10%" style="text-align: left;">Added by</th>
           </tr>
-      </tbody>
-    </table>
-    <button class="button positive" name="action" value="remove">Unblacklist Selected Domains</button>
-  </form>
+        </thead>
+        <tbody>
+          $for item in entries:
+            <tr>
+              <td><input class="checkbox" type="checkbox" name="remove_selection" value="$item['id']"></td>
+              <td>$item['domain_name']</td>
+              <td>$item['reason']</td>
+              <td>$item['name']</td>
+            </tr>
+        </tbody>
+      </table>
+      <button class="button positive" name="action" value="remove">Unblacklist Selected Domains</button>
+    </form>
+  $else:
+    <span class="label" style="text-align: center;">No email domains are currently blacklisted from creating an account.</span>
```

Comment:
`<span class="label">` is for things that label inputs but shouldn’t be `<label>`s for some reason. This can be a `<p>`.